app/src/main/java/com/kickstarter/viewmodels/ProjectHolderViewModel.java

```diff
@@ -83,6 +84,9 @@
     /** Emits the goal string for display. */
     Observable<String> goalStringForTextView();
 
+    /** Emits a boolean for the horizontal rewards feature flag */
+    Observable<Boolean> isHorizontalRewardsEnabled();
```

Comment:
let's rename this to what the outcome of the output is so something like `shouldShowProjectActionButtons`